### ISSUE TITLE
Tunnel: Unknown host

### ISSUE THREAD

**Issue by marek22k** [meshtastic/python#468]
```
$ sudo meshtastic --tunnel
Connected to radio
INFO file:tunnel.py __init__ line:83 Starting IP to mesh tunnel (you must be root for this *pre-alpha* feature to work).  Mesh members:
INFO file:tunnel.py __init__ line:95 Node !5687b499 has IP address None.180.153
INFO file:tunnel.py __init__ line:95 Node !148bfdc5 has IP address None.253.197
None.180.153: Unknown host
ifconfig: `--help' gives usage information.
Aborting due to: ifconfig command failed.
```

Workaround:
```
$ sudo meshtastic --tunnel --subnet 10.115
```

**Comment by OpenSource-For-Freedom** [comment]
have you assigned the correct: 
IP
Subnet
Gateway
DNS

To each one? Can you post an "ifconfig" here for each?
And of course is the Firmware up? I have had a tunnel refusal between two nodes running separate versions.

**Comment by marek22k** [comment]
The problem is that the subnet specified as a command line argument is always used. However, if I don't specify a command line argument, None is used as the value and ifconfig is not aware of None. See my [PR](https://github.com/meshtastic/python/pull/469).

**Comment by OpenSource-For-Freedom** [comment]
"The problem is that the subnet specified as a command line argument is always used."

- so we need to create an option for user to select "/24,/29,/30" options? I feel like if the user network is generally NAT/24, should we just select a 255.255.255.0 initially? And then let users wanting a VPN hosted tunnel make their own CLI changes? 

---

"However, if I don't specify a command line argument, None is used as the value and ifconfig is not aware of None."

- I feel soliciting a dynamic subnet off the start, would allow general user config to be successful. I think it's challenging to guess if the user will "bridge" or "NAT" the handoff. So it might be best to setup a continual "NAT" handoff, /24 network in the initial CLI configuration. I hope I'm on track with your thought here.

**Comment by r-tierney** [comment]
Created a PR which fixes the subnet issue here:
https://github.com/meshtastic/python/pull/476

**Comment by r-tierney** [comment]
This has been fixed in release version 2.2.17 to use the new tunnel feature simply update to this latest release
```bash
pip3 install --upgrade meshtastic
```
